phpcs.xml.dist

```diff
@@ -33,4 +33,10 @@
 	<rule ref="Squiz.Commenting.FunctionComment.MissingParamTag">
 		<exclude-pattern>*/inc/class-cli-command.php</exclude-pattern>
 	</rule>
+
+  <!-- Excluded sniffs -->
+  <rule ref="Universal.Operators.DisallowShortTernary.Found">
+    <exclude-pattern>*/inc/*</exclude-pattern>
+    <exclude-pattern>pantheon-sessions.php</exclude-pattern>
```

Comment:
This line isn't explicitly required and could be removed if we want to. It's doubtful we'd ever need short ternaries in the main plugin file.

Reply:
"[exclude short ternaries being disallowed](https://github.com/pantheon-systems/wp-native-php-sessions/pull/237/commits/ca54ea1aaba1cb45cb0c6ddd65dce8dab673550c)" what a roundabout way of saying "allow short ternaries" 😁

Reply:
I _guess_ you could describe it that way 🙄 😂